manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -320,6 +321,10 @@ data:
             {{ if (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces`) -}}
             - "-k"
             - "{{ index .ObjectMeta.Annotations `traffic.sidecar.istio.io/kubevirtInterfaces` }}"
+            {{ end -}}
+             {{ if (isset .ObjectMeta.Annotations `traffic.sidecar.istio.io/excludeInterfaces`) -}}
+            - "-k"
```

Comment:
is `k` correct? seems its the same as used above?

Reply:
> seems its the same as used above?

Yes, what could be the right one? can you point me to the docs or information about details of which one to use?

Reply:
Fixed. PTAL